applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_quasistatic_U_Pw_scheme.hpp

```diff
@@ -514,21 +512,10 @@ class NewmarkQuasistaticUPwScheme : public Scheme<TSparseSpace,TDenseSpace>
 
 protected:
 
-    /// Member Variables
-
     double mBeta;
     double mGamma;
     double mTheta;
```

Comment:
I would prefer to give these default values (based on what used to be set in the default constructor), just so we can't forget to initialize them (in particular when new constructors are added).